minitrace.c

```diff
@@ -277,7 +277,7 @@ void mtr_flush() {
 		}
 #endif
 
-		len = snprintf(linebuf, ARRAY_SIZE(linebuf), "%s{\"cat\":\"%s\",\"pid\":%i,\"tid\":%i,\"ts\":%llu,\"ph\":\"%c\",\"name\":\"%s\",\"args\":{%s}%s}",
```

Comment:
Does %llu (or %lld) cause problems on Linux? On Windows we really must specify long long because a long is 32-bit.

Reply:
First off: raw->ts is signed, not unsigned, so llu does not seem correct.

With llu:
minitrace.c:280:48: warning: format ‘%llu’ expects argument of type ‘long long unsigned int’, but argument 8 has type ‘int64_t {aka long int}’ [-Wformat=]

With lld:
minitrace.c:280:48: warning: format ‘%lld’ expects argument of type ‘long long int’, but argument 8 has type ‘int64_t {aka long int}’ [-Wformat=]

To fix for both 32b and 64b, PRId64 should be used.
https://stackoverflow.com/a/9225648/301166

I'll try to change this.
Not sure how to adapt a pull request though.
Do I simply add a commit, or something? I'll try.

Reply:
Right. I dislike those macros but I guess they are the right solution here.

You could have just added a commit without closing this one and opened a new one, but that works too.

Anyway, thanks!